packages/labs/context/rollup.config.js

```diff
@@ -10,5 +10,10 @@ import {createRequire} from 'module';
 export default litProdConfig({
   packageName: createRequire(import.meta.url)('./package.json').name,
   entryPoints: ['index'],
-  external: ['@lit/reactive-element', 'lit', 'lit/directive.js'],
+  external: [
+    '@lit/reactive-element',
```

Comment:
Can we make all bare modules external? Bundled is the exception, right? Just polyfill-support?

Reply:
I would have thought that the reactive element entry would have taken care of this. I'd like to look into it more.